src/lib/y2network/udev_rule.rb

```diff
@@ -60,17 +60,30 @@ class << self
       # Returns all persistent network rules
       #
       # @return [Array<UdevRule>] Persistent network rules
-      def all(group = :net)
-        @all ||= {}
-        @all[group] ||= find_rules(group)
+      def all
+        naming_rules + drivers_rules
+      end
+
+      # Returns naming rules
+      #
+      # @return [Array<UdevRule>] Naming network rules
+      def naming_rules
+        find_rules(:net)
+      end
+
+      # Returns driver rules
+      #
+      # @return [Array<UdevRule>] Drivers rules
+      def drivers_rules
+        find_rules(:drivers)
       end
 
       # Returns the udev rule for a given device
       #
       # @param device [String] Network device name
       # @return [UdevRule] udev rule
       def find_for(device)
-        all.find { |r| r.device == device }
+        naming_rules.find { |r| r.device == device }
```

Comment:
here should be documented that it is only naming rule. Not driver rule for given device